assets/training/finetune_acft_image/components/finetune/hf_classification/spec.yaml

```diff
@@ -0,0 +1,352 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+version: 0.0.6
+name: transformers_image_classification_finetune
+display_name: Image Classification HuggingFace Transformers Model Finetune
+description: Component to finetune HuggingFace transformers models for image classification.
+
+is_deterministic: false
+
+environment: azureml://registries/azureml/environments/acft-transformers-image-gpu/versions/1
```

Comment:
Why are we using environments from azureml registry directly, instead of environments from corresponding registries (preview -> preview, staging-> staging) ? Is it because of the restriction that GitHub environments cannot be pushed to azureml-preview registry ?

Reply:
Yes, same as what we did for dnn-vision env.

Reply:
Thanks for confirming